src/imagealternatetext/ui/textareaview.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module image/imagealternatetext/ui/textareaview
+ */
+
+import View from '@ckeditor/ckeditor5-ui/src/view';
+import Template from '@ckeditor/ckeditor5-ui/src/template';
+import '../../../theme/textareaview/theme.scss';
+
+/**
+ * The TextAreaView class.
+ *
+ * @extends module:ui/view~View
+ */
+export default class TextAreaView extends View {
```

Comment:
Yeah, let's use an input. We already have `LabeledInputView` for that kind of stuff in ui.